### PR TITLE
curve bs list snapshot

### PR DESCRIPTION
**Pull request by ZackSoul**
### What problem does this PR solve?

Issue Number: #2832

Problem Summary:

### What is changed and how it works?
add bs list snapshot command

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Wine93** [comment]
cicheck

**Comment by ZackSoul** [comment]
call for review

**Review by caoxianfei1 — APPROVED** [review]

**Comment by caoxianfei1** [comment]
LGTM!

File: tools-v2/pkg/config/bs.go [hunk 1]
```diff
@@ -142,6 +142,8 @@ const (
 	VIPER_CURVEBS_DEST                = "curvebs.dest"
 	CURVEBS_TASKID                    = "taskid"
 	VIPER_CURVEBS_TASKID              = "curvebs.taskid"
+	CURVEBS_SNAPSHOT_ID               = "snapshotid"
+	VIPER_CURVEBS_SNAPSHOT_ID         = "curvebs.snapshotqid"
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/config/bs.go:146]: option flag need default value

**Comment by ZackSoul** [comment]
cicheck

**Comment by caoxianfei1** [comment]
@ZackSoul squash to one commit.

**Review by Cyber-SiKu — APPROVED** [review]

**Pull request merged**